src/components/ThemeProvider/__stories__/ThemeProvider.stories.mdx

```diff
@@ -0,0 +1,166 @@
+Product theme is a custom theme that can be provided by a user, there you can specify the values of specific color tokens for each of the system themes.
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      theme: {
+        name: "overview-theme",
+        colors: {
+          [SystemTheme.LIGHT]: {
+            [ThemeColor.primaryColor]: "green",
+            [ThemeColor.primaryHoverColor]: "darkgreen"
+          },
+          [SystemTheme.DARK]: {
+            [ThemeColor.primaryColor]: "red",
+            [ThemeColor.primaryHoverColor]: "darkred"
+          },
+          black: {
+            "primary-color": "orange",
+            "primary-hover-color": "darkorange"
+          },
+          hacker: {
+            "primary-color": "purple",
+            "primary-hover-color": "darkmagenta"
+          }
+        }
+      }
+    }}
+  >
+    {ThemeProviderTemplateOverview.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    <>
+      Control themes in your application by setting theme classes (e.g. <code>.light-app-theme</code>) on your{" "}
+      <code>body</code> and render everything else inside it
+    </>,
+    <>
+      In most common case ThemeProvider should be rendered only once on the root level of the application - below the{" "}
+      <code>body</code>
+    </>,
+    <>
+      ThemeProvider is populating theme name <code>className</code> to it's child, so don't put{" "}
+      <code>{"\<Fragment\>"}</code> (<code>{"\<\>"}</code>) inside - it's not accepting <code>className</code> prop
+    </>
+  ]}
+/>
+
+<Tip title="Dev tip">
+  Use <code>ThemeProvider.systemThemes</code> and <code>ThemeProvider.colors</code> enums to unleash the power of
+  auto-completion
+</Tip>
+
+## Variants
+
+There are four system themes <code>light</code>, <code>dark</code>, <code>black</code> and <code>hacker</code> and in each you can redefine the values of the following color tokens:
+
+### Colors eligible for theming
+
+<Story name="Colors eligible for theming">
+  <ColorsEligibleForTheming />
+</Story>
+
+## Use cases and examples
+
+### Theming scope
+
+Only components wrapped with ThemeProvider will be affected by the theme.
+
+<Canvas>
+  <Story name="Theming scope">{ThemeProviderThemingScopeTemplate.bind({})}</Story>
+</Canvas>
+
+### Folded theming
+
+If component is wrapped with multiple ThemeProviders, the most nested one will override the values of the outer one, but if the nested ThemeProvider doesn't provide a value for a specific color token, the outer ThemeProvider will be used.
+
+<Canvas>
+  <Story name="Folded theming">{ThemeProviderFoldedThemingTemplate.bind({})}</Story>
+</Canvas>
+
+### Product theming
+
+These are theme-definitions, which are used in monday.com products.
+
+<Canvas>
+  <Story name="Product theming">{ThemeProviderProductThemingTemplate.bind({})}</Story>
```

Comment:
<img width="1022" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/fb025df9-6fed-4bc1-b175-69d1a98973a6">
not working

Reply:
Fixed